pylint_odoo/misc.py

```diff
@@ -112,14 +113,27 @@ def get_manifest_file(self, node_file):
 
     def set_ext_files(self):
         """Create `self.ext_files` dictionary with {extension_file: [files]}
+            and exclude files using --ignore and --ignore-patterns parameters
         """
         self.ext_files = {}
+        ignores = list(self.linter.config.black_list)
```

Comment:
Why we need this variable instead of use it directly?
NOTE: I requested a explanation, don't confuse with a request of change.